boards/arm/stm32h7/nucleo-h743zi/Kconfig

```diff
@@ -27,4 +27,66 @@ config STM32_ROMFS_IMAGEFILE
         depends on STM32_ROMFS
         default "../../../rom.img"
 
+
+config PROGMEM_OTA_PARTITION
+	bool "Progmem ota partitions"
+	default y
+	select MTD
+	select MTD_BYTE_WRITE
+	select MTD_PARTITION
+	select MTD_PROGMEM
+	select MTD_PROGMEM_ERASESTATE
+
+menuconfig APP_FORMAT_MCUBOOT
+	bool "MCUboot-bootable format"
+	default y
+	select PROGMEM_OTA_PARTITION
+	---help---
+		The MCUboot support of loading the firmware images.
+
+if APP_FORMAT_MCUBOOT
+
+config MCUBOOT_BOOTLOADER
+	bool "MCUboot bootloader application"
+	default y
+	---help---
+		This switch between linker scripts to allow an application be
+		built to another entry point address.
+
+comment "MCUboot Application Image OTA Update support"
+
+config OTA_PRIMARY_SLOT_DEVPATH
+	string "Application image primary slot device path"
+	default "/dev/ota0"
+
+config OTA_SECONDARY_SLOT_DEVPATH
+	string "Application image secondary slot device path"
+	default "/dev/ota1"
+
+config OTA_SCRATCH_DEVPATH
+	string "Scratch partition device path"
+	default "/dev/otascratch"
+
+config OTA_PRIMARY_SLOT_OFFSET
+	hex "MCUboot application image primary slot offset"
+	default "0x20000"
+
+config OTA_SECONDARY_SLOT_OFFSET
+	hex "MCUboot application image secondary slot offset"
+	default "0x100000"
```

Comment:
Please check here. I see that `OTA_PRIMARY_SLOT_OFFSET`+`OTA_SLOT_SIZE` -> `0x20000 + 0xc0000 == 0xe0000` so there still is `0x20000` free room left

Reply:
Yeah I know. If I use the entire memory of flash the MTD device fails writing. There must be a BUG in stm32_flash.c file when trying to work with the last sectors of each flash bank.

I will try to solve it ASAP